mappings/MGIAlzheimerMP_HPO_mapping.tsv

```diff
@@ -0,0 +1,23 @@
+subject_id	subject_label	predicate_id	object_id	object_label	match_type	sssom:confidence	sssom:creator_id	sssom:mapping_date	sssom:comment
+MP:0003329	amyloid beta deposits	skos: closeMatch	HP:0100256	Senile plaques	HumanCurated	9	orcid:0000-0003-2307-1226	2021 - 07 - 29	
```

Comment:
subject, object, predicate, creator id fields:

- must not contain spaces (i.e. be valid CURIEs), so you cannot have `skos: exactMatch` (must be `skos:exactMatch`).
- all prefixes (skos, HP, MP, orcid, PMID, etc) must be declared in the curie_map, see example above.